### DISCUSSION TITLE
License has expired, but I just bought it on 12/07/2024

### DISCUSSION THREAD

**Discussion by wilf974** [rustdesk/rustdesk-server-pro#310] [Q&A] [answered]
Hello, I have just acquired a license for 10 users and 200 workstations which has worked well until now. It has expired although I paid for 1 year. This is a handicap.

**Comment by rustdesk** [comment]
Apologize, let me check, I just update server program (but forgot to migrate datebase), it might have problem, let me revert back first.

**Reply by wilf974** [reply]
Ok thank's

**Reply by wilf974** [reply]
What information do you need?

**Reply by rustdesk** [reply, marked as answer]
Could you try again? goto web console, `Settings` -> `License`, click on `Edit` button, and then click on `Confirm` button directly.

**Reply by wilf974** [reply]
Thank you, it worked. Thank you for your efficiency :)

**Reply by rustdesk** [reply]
what's your email used for the license?

**Reply by wilf974** [reply]
exploitit@divabox.net

**Reply by wilf974** [reply]
in fact it tells me that the license is already being used by another machine

**Reply by rustdesk** [reply]
Are you running docker? have you enabled `--net=host`? https://rustdesk.com/docs/en/self-host/rustdesk-server-pro/installscript/docker/#:~:text=uses%20sudo%20and-,%2D%2Dnet%3Dhost,-%2C%20this%20will%20not

**Reply by wilf974** [reply]
docker desktop

**Reply by rustdesk** [reply]
Please enable `--net=host`.

**Reply by wilf974** [reply]
i have this :    volumes:
      - ./data:/root
    networks:
      - rustdesk-net
    depends_on:
      - hbbr

**Reply by wilf974** [reply]
i replace : rustdesk-net to "host" ?

**Reply by rustdesk** [reply]
https://rustdesk.com/docs/en/self-host/rustdesk-server-pro/installscript/docker/#docker-compose

![image](https://github.com/user-attachments/assets/af2bb789-0092-47bd-9b09-b9602c0be78c)

**Reply by wilf974** [reply]
![image](https://github.com/user-attachments/assets/dc4fc11a-3ca8-4d8e-a6a5-5744f32c1209)

**Reply by rustdesk** [reply]
Please do this first, https://rustdesk.com/docs/en/self-host/rustdesk-server-pro/license/#invoices-license-retrieval-and-migration

**Reply by wilf974** [reply]
Thank's for your help :)
